src/plugins/autoplay/autoplay.js

```diff
@@ -10,6 +10,8 @@
     "use strict";
 
     var autoplayDefault = 0;
+    var userSkipAutoplayDefault = 0;
```

Comment:
It looks like I was lazy and there's no documentation for this plugin. (Either in the comment above, or a separate README file.) Could you add it now? Including your own addition.